Dockerfile

```diff
@@ -2,12 +2,18 @@
 # Use of this source code is governed by an Apache2
 # license that can be found in the LICENSE file.
 
-FROM scratch
+FROM alpine:latest
 
 MAINTAINER Torin Sandall <torinsandall@gmail.com>
 
-ADD /opa_linux_GOARCH /opa
+ENV BIN /usr/bin/opa
```

Comment:
I don't know, but I also don't think it matters much because in a docker world, you can control a lot of system things and don't have to worry much about other tasks (cron, package updates, etc.) messing you up. I am sure people are more passionate about where to put it and really don't mind where it goes. I just liked it in the `$PATH` :).